tools/docker/image-pull.sh

```diff
@@ -17,7 +17,6 @@ usage() {
     echo "      -h|--help - show this help menu"
     echo "      -v|--verbose - verbosity on"
     echo "      -b|--base-image - Base OS image to use (Dockerfile 'FROM')"
-    echo "      -n|--native - Use the same base OS image as the running system"
```

Comment:
Please drop this commit or fix the --native option - why isn't it usable? if indeed there is a bug, it should be fixed rather than removed.

Reply:
> Please drop this commit or fix the --native option - why isn't it usable?

That was my suggestion. Obviously, the explanation should have been part of the commit message.

```
The --native option doesn't really work because:
- for starters, it only applies to Debian derivatives, because the Dockerfile uses APT;
- in addition, it only works with Ubuntu because the package names differ slightly;
- finally, it only works with Ubuntu 18.04 and a few versions very close to it, because of package incompatibilities.

In other words, we will anyway need a separate Dockerfile for any platform that is not Ubuntu 18.04 (or close to it, e.g. 18.10 does work). Therefore the --native option is next to useless.
```

Unless you know someone who actually uses this option, @tomereli ?

Reply:
All of Intel users are working on Ubuntu18.04 LTS server, so we used to use the -n option all the time. I haven't used it in a while since I didn't need to, so I'm not strictly against removing it - but I prefer to keep it unless its too much work.

Reply:
On an Ubuntu 18.04 the -n option is a NOP, it will set IMAGE to its default value `ubuntu:18.04`.

Reply:
So it was probably used in the beginning when we used Ubuntu19 in our HyperV VMs - in that case I guess we can remove this.